packages/terra-alert/src/Alert.module.scss

```diff
@@ -12,6 +12,7 @@
     margin: 0 0 1px;
     min-height: var(--terra-alert-min-height);
     overflow: hidden;
+    overflow-wrap: anywhere;
```

Comment:
I tried to find a way to get it working with `overflow-wrap: break-word` and did find that it works if `min-width: 0` (or otherwise a very small width) is set on the [alert section](https://github.com/cerner/terra-core/blob/810b7a1e8b643ab84ef56d713ecc4cd29591a352/packages/terra-alert/src/Alert.module.scss#L58). However, the title does not also wrap and needs to have `width: 100%` set as well.

It seems like the difference in behavior is due to the use of [`display: flex`](https://github.com/cerner/terra-core/blob/810b7a1e8b643ab84ef56d713ecc4cd29591a352/packages/terra-alert/src/Alert.module.scss#L10). `overflow-wrap: anywhere` (as shown in the screenshots) does wrap the text at very narrow widths, while `overflow-wrap: break-word` has no effect on its own. ([couple](https://stackoverflow.com/questions/36150458/flex-item-overflows-container-due-to-long-word-even-after-using-word-wrap) [links](https://dev.to/somshekhar/overflow-wrap-in-a-flex-container-35))

Is there a concern with using `anywhere` instead of `break-word`?